config/services.yaml

```diff
@@ -79,6 +79,13 @@ services:
     public: true
   App\Catrobat\Security\ApiKeyAuthenticator: "@apikey_authenticator"
 
+  webview_authenticator:
+    class: App\Catrobat\Security\WebviewAuthenticator
+    arguments:
```

Comment:
arguments section is not needed; Dependency injection of TranslatorInterface in the constructor of webview_authenticator is already enough.